src/AppServices/KafkaPage/KafkaPage.tsx

```diff
@@ -35,50 +50,208 @@ export const KafkaPage: FunctionComponent = () => {
+                href="https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-streams-for-apache-kafka#contact-us"
                 target="_blank"
               >
-                {t("kafkaoverview:contactSalesCardCallToActionButton")}
+                {t("contactSalesCardCallToActionButton")}
                 <ExternalLinkAltIcon className="pf-u-ml-md" />
               </Button>
             </CardFooter>
           </Card>
         </Grid>
       </MarketingPageSection>
+      <MarketingPageSection>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h3">{t("pricingModalTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Flex spaceItems={{ default: "spaceItemsXl" }}>
+              <Flex
+                alignSelf={{ default: "alignSelfCenter" }}
+                justifyContent={{ default: "justifyContentCenter" }}
+                flex={{ default: "flex_1" }}
+              >
+                <FlexItem>
+                  <DescriptionList isHorizontal>
+                    <DescriptionListGroup>
+                      <DescriptionListTerm>
+                        {t("steamingUnit")}
+                      </DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$1.49/hour"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                    <DescriptionListGroup>
+                      <DescriptionListTerm>
+                        {t("dataTransfer")}
+                      </DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$0.09/GB"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                    <DescriptionListGroup>
+                      <DescriptionListTerm>{t("storage")}</DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$0.10/GB"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                  </DescriptionList>
+                </FlexItem>
+              </Flex>
+              <Divider isVertical />
+              <FlexItem flex={{ default: "flex_1" }}>
+                <TextContent>
+                  <Text component={TextVariants.p}>
+                    <Trans
+                      i18nKey={"kafkaoverview:streamingUnitText"}
+                      values={{
+                        value: "streaming unit",
+                      }}
+                    />
+                  </Text>
+                </TextContent>
+              </FlexItem>
+            </Flex>
+          </CardBody>
+        </Card>
+      </MarketingPageSection>
+      <MarketingPageSection>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h3">{t("kafkaInstanceCapacityTitle")}</Title>
```

Comment:
I see there's an `<h1>` element but not an `<h2>`. All of the card titles should be updated to use `headingLevel="h2"`